lib/tasks/evm_automate.rake

```diff
@@ -148,14 +148,26 @@ namespace :evm do
         elsif ENV['YAML_FILE'].present?
           puts "Importing automate domain: #{ENV['DOMAIN']} from file #{ENV['YAML_FILE']}"
           import_options['yaml_file']   = ENV['YAML_FILE']
+        elsif ENV['GIT_URL'].present?
+          puts "Importing automate domain from url #{ENV['GIT_URL']}"
+          import_options['url'] = ENV['GIT_URL']
+          import_options['overwrite'] = true
+          import_options['userid'] = ENV['USERID']
+          import_options['password'] = ENV['PASSWORD']
+          import_options['ref'] = ENV['REF'] || MiqAeDomain::DEFAULT_BRANCH
+          import_options['ref_type'] = ENV['REF_TYPE'] || MiqAeDomain::BRANCH
         end
         %w(SYSTEM ENABLED).each do |name|
           if ENV[name].present?
             raise "#{name} must be true or false" unless %w(true false).include?(ENV[name])
             import_options[name.downcase] = ENV[name]
           end
         end
-        MiqAeImport.new(ENV['DOMAIN'], import_options).import
+        if ENV['GIT_URL'].present?
+          MiqAeDomain.import_git_url(import_options)
+        else
+          MiqAeImport.new(ENV['DOMAIN'], import_options).import
+        end
```

Comment:
I don't like this structure.  The beauty of a class like MiqAeImport is that it has a very clear duty.  Now we've cluttered MiqAeDomain with an import method *and* we have a class for importing.  Instead, I think that MiqAeImport should have two class-level interface methods.  `MiqAeImport.import_git` (no need for the URL, because it's implied) and `MiqAeImport.import` (not sure you need any qualifier here).  This method would then just call one of the two import methods.  In fact, both could just take import_optoins and the domain could just be one of the options, making the interface consistent.

Under the covers, `import_git` should call can MiqAeImportGit class that does all the git based importing, and `import` should call a separate MiqAeImport<Something> class that does the other kind of import.